### PR TITLE
docs: Update changelog and podspec for v2.1.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Preparation for 2.1.0 release.

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

File: .travis.yml [hunk 1]
```diff
@@ -85,9 +85,8 @@ after_success:
 deploy:
   # Deploy Docs to Github Pages when tagged as release
   provider: pages
-  strategy: git
-  cleanup: false
-  token: $GITHUB_TOKEN
+  skip_cleanup: true
```
Comment (Climbatize) [inline .travis.yml:88]: `skip_cleanup` is deprecated, that's why I used `cleanup` lest time

File: .travis.yml [hunk 2]
```diff
@@ -85,9 +85,8 @@ after_success:
...
+  github_token: $GITHUB_TOKEN
```
Comment (Climbatize) [inline .travis.yml:89]: same for `GitHub_token`

**Review by Climbatize — APPROVED** [review]
LGTM, be careful to deprecations

File: .travis.yml (hunk 1 above)
Comment (corycaywood) [reply .travis.yml:88]: Yeah, it's strange. TravisCI gives a warning when you use this setting, but using just `cleanup` doesn't work. See the last release log: https://travis-ci.org/github/rakutentech/ios-miniapp/builds/715699238

The docs for the pages deployment still have the old values, so I think they haven't updated this plugin yet: https://docs.travis-ci.com/user/deployment/pages/#further-configuration

File: .travis.yml (hunk 2 above)
Comment (corycaywood) [reply .travis.yml:89]: Same for this one, it seems the github token wasn't working in the last release: : https://travis-ci.org/github/rakutentech/ios-miniapp/builds/715699238

**Pull request merged**